wire-ios-data-model/Source/Model/User/ZMUser.swift

```diff
@@ -452,11 +452,7 @@ extension ZMUser: UserConnections {
             case .success:
                 Task {
                     do {
-                        guard let resolver = oneOnOneResolver ?? OneOnOneResolver(syncContext: syncContext) else {
-                            completion(AcceptConnectionError.unableToResolveConversation)
-                            return
-                        }
-
+                        let resolver: OneOnOneResolverInterface = oneOnOneResolver ?? OneOnOneResolver(syncContext: syncContext)
```

Comment:
suggestion: delete the explicit type declaration if you can.

Reply:
Done!